gnmi_cli_py/py_gnmicli.py

```diff
@@ -275,6 +300,13 @@ def _get_val(json_value):
       raise JsonReadError('Error while loading JSON: %s' % str(e))
     val.json_ietf_val = json.dumps(set_json).encode()
     return val
+  elif '$' in json_value:
+    try:
+      proto_bytes = six.moves.builtins.open(json_value.strip('$'), 'rb').read()
+    except (IOError, ValueError) as e:
+      raise Exception('Error while loading proto: %s' % str(e))
```

Comment:
>Exception [](http://example.com/codeflow?start=12&length=9)

Suggest to use `ValueError`. Does the error message contains the filename? If not, suggest add it to help troubleshoot.

Reply:
Updated.

Reply:
updated.